openwisp_controller/config/settings.py

```diff
@@ -52,3 +52,4 @@ def get_settings_value(option, default):
 }
 HARDWARE_ID_OPTIONS.update(get_settings_value('HARDWARE_ID_OPTIONS', {}))
 HARDWARE_ID_AS_NAME = get_settings_value('HARDWARE_ID_AS_NAME', True)
+OPENWISP_DEVICE_VERBOSE_NAME = 'device'
```

Comment:
Please follow the way other settings have been defined. This could be written as `DEVICE_VERBOSE_NAME = get_settings_value('DEVICE_VERBOSE_NAME',  'Device')

Reply:
ok, @NoumbissiValere  I will update the readme as well as do the mentioned changes.
Thanks.

Reply:
Also, i want to know how to pass these QA-checks because while running locally it is passing but in Travis, it is failing.

Reply:
You can do that with the command `./run-qa-checks`